requirements.txt

```diff
@@ -37,4 +37,5 @@ anthropic==0.3.6
 typing-inspect==0.8.0
 typing_extensions==4.5.0
 libcst==1.0.1
-qdrant-client==1.4.0
\ No newline at end of file
+qdrant-client==1.4.0
+pytest-mock==3.11.1
```

Comment:
The pytest-mock is a development dependency. Perhaps we should create a requirements-dev.txt file?

Reply:
then pytest is also a development dependency. What do you suggest? Perhaps another pr.
but our packaging mechanism hasn't been decided yet.